### DISCUSSION TITLE
how can i set Calico ENV "CALICO_IPV4POOL_BLOCK_SIZE" in windows node?

### DISCUSSION THREAD

**Discussion by oingooing** [rancher/rke2#4144] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

hi
i want to use Calico Env Option that CALICO_IPV4POOL_BLOCK_SIZE. (default : 26)
i know that kube-proxy currently only supports a single IP block per host in windows. (https://docs.tigera.io/calico/3.24/getting-started/kubernetes/windows-calico/limitations#calico-vxlan-networking-limitations)

As an alternative, try increasing the ip block size. (/20)

i cannot find calico env configuration in rke2 config file. 
how can i set calico env? persists even when the node is restarted.

thanks.

**Describe the solution you'd like**

It would be nice if rke2 didn't overwrite the settings on node restart
or
possible input with rke2 settings

**Additional context**

**Rancher Version** : v2.6.9
**K8s Version** : v1.24.10+rke2r1
**Calico Version** : 3.24.5
**Windows Node OS** : Windows Server 2019 STD

**Comment by brandond** [comment, marked as answer]
You probably shouldn't be reading the upstream Tigera docs on how to install Windows agents, as we do not use any of their powershell scripts to manage the CNI installation. Please refer to the RKE2 docs, and use the RKE2 powershell install script. Do NOT manually install any of the components using instructions from Tigera. This includes trying to set Calico environment variables on Windows; theses variables will be managed by the helm chart bundled with RKE2. If you want to customize the IP pool CIDR size, you can do that by providing a custom IPPool value to the rke2-calico helm chart config.
